client/src/index.jsx

```diff
@@ -14,13 +15,34 @@ const Wrapper = Styled.div`
 class App extends React.Component {
   constructor() {
     super();
+    
+    this.state = {
+      locations: [],
+      materials: []
+    }
+    this.getProductById = this.getProductById.bind(this);
   }
   
+  getProductById() {
+    let id = window.location.pathname;
+    axios.get(`/locations${id}`)
+      .then(response => this.setState({ locations: response.data }))
+      .catch(error => console.log(error))
+    axios.get(`/materials${id}`)
+      .then(response => this.setState({ locations: response.data }))
```

Comment:
you're overwriting locations with materials information here

Reply:
lol! Yeah, I fixed this one in the next one.